rootfs/api/models/app.py

```diff
@@ -538,7 +539,11 @@ def list_pods(self, *args, **kwargs):
                 item['state'] = self._scheduler.resolve_state(p).name
                 item['release'] = p['metadata']['labels']['version']
                 item['type'] = p['metadata']['labels']['type']
-                item['started'] = p['status']['startTime']
+                if 'startTime' in p['status']:
+                    started = p['status']['startTime']
+                else:
+                    started = str(datetime.utcnow().strftime('%Y-%m-%dT%H:%M:%SZ'))
```

Comment:
we should use `settings.DEIS_DATETIME_FORMAT` so we can change this if necessary: https://github.com/deis/workflow/blob/master/rootfs/deis/settings.py#L148